.env.example

```
@@ -0,0 +1,30 @@
+# Stargate V2 Endpoints
+ENDPOINT_ETHEREUM=0x1a44076050125825900e736c501f859c50fE728c
+ENDPOINT_BASE=0x1a44076050125825900e736c501f859c50fE728c
+ENDPOINT_BNB=0x1a44076050125825900e736c501f859c50fE728c
+ENDPOINT_AVALANCHE=0x1a44076050125825900e736c501f859c50fE728c
+ENDPOINT_POLYGON=0x1a44076050125825900e736c501f859c50fE728c
+ENDPOINT_ARBITRUM=0x1a44076050125825900e736c501f859c50fE728c
+ENDPOINT_OPTIMISM=0x1a44076050125825900e736c501f859c50fE728c
+
+# RPC Endpoints
+ETHEREUM_RPC_URL=https://rpc.ankr.com/eth
```

```suggestion
ETHEREUM_RPC_URL=https://rpc.ankr.com/eth
```

use our internal domain mapping to `{{chain}}.gemnodes.com`, no need to include RPC here.